docker/yb-voyager-docker

```diff
@@ -27,29 +45,49 @@ for var_name in "${variables[@]}"; do
   fi
 done
 
-exported_vars="${exported_vars} -e ORACLE_HOME=/usr/lib/oracle/21/client64 -e LD_LIBRARY_PATH=/usr/lib/oracle/21/client64/lib -e PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/usr/local/go/bin:/usr/lib/oracle/21/client64/bin"
-
-while [ $i -lt $# ]
-do
-	if [[ ${argv[${i}]} == "--export-dir" || ${argv[${i}]} == "-e" ]]
-	then
-		j=$(( $i + 1))
-		export_dir=${argv[${j}]}
-		export_dir=$(realpath "$export_dir")
+exported_vars="${exported_vars} -e LC_ALL=en_US.UTF-8 -e LC_CTYPE=en_US.UTF-8 -e ORACLE_HOME=/usr/lib/oracle/21/client64 -e LD_LIBRARY_PATH=/usr/lib/oracle/21/client64/lib -e PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/usr/local/go/bin:/usr/lib/oracle/21/client64/bin"
 
+map_host_path_inside_container() {
+		flag_name=$1
+		dir=$2
```

Comment:
Looks like `dir` is a name of an existing command from coreutils. Consider changing this name to `dir_path`.